subsys/bluetooth/shell/bt.c

```diff
@@ -3894,9 +3894,12 @@ static int cmd_auth_passkey_notify(const struct shell *sh,
 
 	err = 0;
 	type = shell_strtoul(argv[1], 0, &err);
-	if (err || !IN_RANGE(type, BT_CONN_AUTH_KEYPRESS_ENTRY_STARTED,
-			     BT_CONN_AUTH_KEYPRESS_ENTRY_COMPLETED)) {
-		shell_error(sh, "<type> must be a value in range of enum bt_conn_auth_keypress");
+	if (err ||
+	    (type != BT_CONN_AUTH_KEYPRESS_ENTRY_STARTED &&
+	     type != BT_CONN_AUTH_KEYPRESS_DIGIT_ENTERED &&
+	     type != BT_CONN_AUTH_KEYPRESS_DIGIT_ERASED && type != BT_CONN_AUTH_KEYPRESS_CLEARED &&
```

Comment:
I think the below is more readable
```suggestion
	     type != BT_CONN_AUTH_KEYPRESS_DIGIT_ERASED && 
	     type != BT_CONN_AUTH_KEYPRESS_CLEARED &&
```

Reply:
The original is from clang format. Do you have an idea for an alternative that autoformats nicely?